server/api/replies.post.ts

```diff
@@ -0,0 +1,44 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export default defineEventHandler(async (event) => {
+  try {
+    const body = await readBody(event);
+    const pageCuid = body.pageCuid;
+    const familyCuid = body.familyCuid;
+
+    const page = await prisma.page.findFirst({
+      where: {
+        cuid: pageCuid,
+      },
+    });
+
+    if (!page) {
+      console.error(`Page with cuid ${pageCuid} not found`);
+      return { status: "error", message: "Page not found" };
+    }
+
+    const newReply = await prisma.reply.create({
+      data: {
+        reply: body.replyData.reply,
+        name: body.replyData.name,
```

Comment:
use spread here: `data: {...replyData, Page, familyCuid}` - you don't have to pluck out the properties of `replyData` individually, you can just splat the object in